SetGame/ViewController.swift

```diff
@@ -39,6 +39,7 @@ class ViewController: UIViewController {
 	func updateUI() {
 		for i in vm.cardsOnTable.indices {
 			buttons[i].isHidden = false
+			buttons[i].setTitle(vm.cardsOnTable[i].text, for: .normal)
```

Comment:
It could also be in a separate commit. It does not fit the current commit message!